packages/devextreme/testing/tests/DevExpress.ui.widgets/toolbar.menu.tests.js

```diff
@@ -595,6 +599,85 @@ QUnit.module('widget sizing render', moduleConfig, () => {
 
         });
 
+        QUnit.test('space on list item hides drop-down list (T1301705)', function(assert) {
+            if(shouldSkipOnMobile(assert)) {
+                return;
+            }
+
+            this.instance.option('closeOnClick', true);
+            this.instance.option('opened', false);
+            this.overflowMenu.$button().focusin();
+
+            this.keyboard.keyDown('enter');
+            const popup = this.overflowMenu.popup();
+            assert.equal(popup.option('visible'), true, 'popup is visible');
+
+            this.instance.option(
+                'items', [
+                    {
+                        template: () => $('<div>').dxCheckBox({ value: false }),
+                        widget: 'dxCheckBox',
+                    },
+                    {
+                        template: () => $('<div>').dxButton({ text: 'Button' })
+                    },
+                ]
+            );
+
+            const checkBox = this.overflowMenu.$popupContent().find(`.${CHECK_BOX_CLASS}`).first();
+            const button = this.overflowMenu.$popupContent().find(`.${BUTTON_CLASS}`).first();
```

Comment:
We usually use `$` prefix to clearly show it's a jquery element and not a native element or component instance
```suggestion
            const $button = this.overflowMenu.$popupContent().find(`.${BUTTON_CLASS}`).first();
```